### PR TITLE
ext_time_quota_acl: remove -l option

### PR DESCRIPTION
**Pull request by kinkie**
Supporting logging to a file complicates upgrading helper code to use
debugs() because DebugFile code calls commSetCloseOnExec(), and our
comm/libminimal does not currently provide a functioning implementation
for that API: The existing implementation is an unconditional assert.
To save development time while upgrading helpers, we are dropping this
feature. It can probably be emulated using shell redirection tricks.

----
Manual backport of master/v7 commit 2d93cfe7.

### CODE REVIEW

**Comment by kinkie** [comment]
Note: after merging this backport we will need to do a followup PR to
v7 to move the release note content from the v7 to the v6 file

**Comment by rousskov** [comment]
I have adjusted PR description footer to refer to a (more permanent and actually meaningful in some contexts[^1]) branch commit rather than GitHub PR.

[^1]: Imagine, for example, dealing with v6 branch a few months from now, when we have both v7 _and_ master/v8 branches. It is often useful to know where (i.e. which branch) a particular change was backported from. The same "source branch" information can be restored using `git`, but it only takes three extra characters to inline it (by adding "/v7" suffix) and make it readily available to the reader... I do not insist on adding such suffixes, of course. Your call!

@kinkie, I recommend adjusting that footer further to also mention _why_ manual changes were necessary in this case and documenting their perceived risks, divergence from master/v7 code, etc. The reader already knows that cherry-picking failed, but does not know what the side effects of those manual changes were. Let's help the reader understand what they are dealing with!

**Comment by rousskov** [comment]
I also fixed PR description formatting using the original commit message.

**Comment by kinkie** [comment]
ok to test

**Comment by kinkie** [comment]
ok to test

File: doc/release-notes/release-6.sgml.in
Comment (yadij) [inline doc/release-notes/release-6.sgml.in:285]:
```suggestion
	<tag>Adjusted configuration and format of <em>ext_time_quota_acl</em> helper debugging</tag>
```

File: doc/release-notes/release-6.sgml.in
Comment (yadij) [inline doc/release-notes/release-6.sgml.in:288]:
```suggestion
		changed to be in line with Squid <em>cache.log</em> format.
```

**Review by yadij — APPROVED** [review]
Just some polish to the release notes that was missed first time around. Not insisting on a change.

**Comment by yadij** [comment]
Jenkins failing with Java errors. Re-basing to kick a retest and see if they are gone.

**Comment by kinkie** [comment]
> Jenkins failing with Java errors. Re-basing to kick a retest and see if they are gone.

yeah, it happens sometimes; it's race conditions on temporary commits I think

**Comment by yadij** [comment]
FYI, I have not heard anything about Anubis being fixed to handle the vN branch merges yet. So having it test backports is a waste of resources.

**Comment by rousskov** [comment]
> FYI, I have not heard anything about Anubis being fixed to handle the vN branch merges yet.

FWIW, I am not aware of any Anubis bugs in this context. IIRC, Anubis treats all target branches the same -- it does not treat the master branch specially. If you want to use Anubis for merging vN PRs and need Anubis to treat master and those branches differently, please let me know what different treatment you would like to see or file/update the corresponding [issue](https://github.com/measurement-factory/anubis/issues). Maintainers are in the driving seat for these activities.

> So having it test backports is a waste of resources.

Even if a PR is merged manually, testing of [staged commits](https://github.com/measurement-factory/anubis/blob/960addcb60f4de38e051671b6bab059a279e250e/README.md#staging-commit) (created by Anubis) may be valuable: Jenkins and GitHub test PR merge commits which are often slightly different from staged commits. Besides testing, staged commits offer correct commit message (that can be edited by editing PR title/description) and correct code tree changes. Squid vN branches have bad commits (e.g., commit b4addc22, commits 5f6a24fc..23b3ee04, and commit 94bdd490) that would not exist if staged commits were merged (manually or automatically).

**Pull request merged**
